### DISCUSSION TITLE
Restore Plex DB

### DISCUSSION THREAD

**Discussion by EQ5** [hassio-addons/addon-plex#180] [Q&A]
Hello,

I would like to restore the DB to HA Plex from an existing Plex application.
How can I copy the DB to the HA Plex instance?

Regards
Manuel

**Comment by 123gizi** [comment]
Hey,
I would also love to know if there's a way to do this. Is there directory that I can access using SSH to copy to/from for an offline backup of my PMS without the need to reload from scratch using the Addon or a full HA restore?

**Comment by frenck** [comment]
There is no such way supported and/or recommended. Sorry.
